examples/inspection/plot_partial_dependence.py

```diff
@@ -240,24 +261,26 @@
 
 import numpy as np
 from mpl_toolkits.mplot3d import Axes3D
+
 fig = plt.figure()
 
-features = ('AveOccup', 'HouseAge')
+features = ("AveOccup", "HouseAge")
 pdp = partial_dependence(
-    est, X_train, features=features, kind='average', grid_resolution=20
+    est, X_train, features=features, kind="average", grid_resolution=20
 )
 XX, YY = np.meshgrid(pdp["values"][0], pdp["values"][1])
 Z = pdp.average[0].T
 ax = Axes3D(fig)
```

Comment:
To prevent [this `MatplotlibDeprecationWarning`](https://github.com/matplotlib/matplotlib/blob/8c764dc86b6b24fced137105e6f97a5d62d4a717/lib/mpl_toolkits/mplot3d/axes3d.py#L155-L165) from being raised.
```suggestion
ax = Axes3D(fig, auto_add_to_figure=False)
fig.add_axes(ax)
```